src/routes.tsx

```diff
@@ -63,71 +53,74 @@ const navItems: MenuItem[] = [
 
 const defaultRoute = '/overview';
 
+const graphPage = lazy(() => import('./pages/Graph/GraphPage'));
+const istioConfigDetailsPage = lazy(() => import('./pages/IstioConfigDetails/IstioConfigDetailsPage'));
+
 const pathRoutes: Path[] = [
   {
     path: '/overview',
-    component: OverviewPageContainer
+    component: lazy(() => import('./pages/Overview/OverviewPage'))
```

Comment:
Why did you followed a different pattern here? (directly calling lazy here)

Reply:
No hard reason. I just called directly where I saw no need to introduce a variable.
I can use a consistent pattern if that looks better.

Reply:
I like forcing `lazy` when possible.